### PR TITLE
Use latest nvptx-tools to fix builds when CUDA is installed for GCCcore 9.3.0/10.2.0

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

Intended to fix https://github.com/easybuilders/easybuild-easyconfigs/issues/12518

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 0 out of 0 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/0efb467793478c0ca8cb38a52ac4b74c for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/f042710af45d66b023af3cd3df1a8cc0 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Failed for me on a node with CUDA already installed.

**Comment by boegel** [comment]
@verdurin I think this is superseded by #13290, is that correct?

**Comment by boegel** [comment]
@verdurin I've cancelled the test on `generoso` for this PR, since the tests being run for #13290 were clashing with it.

**Comment by verdurin** [comment]
> @verdurin I think this is superseded by #13290, is that correct?

@boegel yes, I think so. Was a bit hard to follow the different suggestions so thought I would try the most recent one. Will close this if the other approach works.

**Comment by boegel** [comment]
@verdurin With #13290 merged, I think we can close this one.

**Comment by verdurin** [comment]
Better fix is in #13290 so closing.
